Python/Tic-Tac-Toe/Tic-tac-toe.py

```diff
@@ -0,0 +1,71 @@
+"""
+@author: Arpit Somani
+"""
+
+import numpy

```

Comment:
- add comments to this program
- add a GIF to demonstrate/demo of the game
- place the GIF in this Tic-Tac-Toe folder
- Edit the readme of the python folder